README.md

```diff
@@ -197,6 +197,9 @@ let g:nv_show_preview = 1
 " Boolean. Respect .*ignore files in or above nv_search_paths. Set by default.
 let g:nv_use_ignore_files = 1
 
+" Boolean. Include hidden files and folders in search. Disabled by default.
+let g:nv_include_hidden = 1
```

Comment:
This should be 0 if it's disabled.

Reply:
OK! (Note that the readme isn't completely consistent about this, see `g:nv_ignore_pattern`, but it makes sense.)